fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1714,6 +1729,13 @@ public TStorageFormat getStorageFormat() {
         return tableProperty.getStorageFormat();
     }
 
+    public DataSortInfo getDataSortInfo() {
+        if (tableProperty == null) {
+            return new DataSortInfo();
```

Comment:
Better assign the default value to the fields of `DataSortInfo`, so that we can make this return more semantically accurate